.github/workflows/package-workflow.yml

```diff
@@ -2,44 +2,36 @@
 on:
   push:
     branches:
-      - master
+    - master
     paths:
-      - "scripts/actions/build-deb/"
-      - ".github/workflows/package-workflow.yml"
+    - scripts/actions/build-deb
```

Comment:
What auto-formatter are you using for this? VS Code reverts back to the old formatting when I save the file. 🤷‍♂️

Reply:
Oh? I use VSCode but it seems I don't have a formatter by default - I was following the style from the other files, but I'm fine if we follow an autofix for YAML files.

Reply:
I wasn't 100% sure where the auto-formatting was coming from, but I think it's coming from Prettier (https://prettier.io/) which appears to support YAML now.